open-problems/brief-problem-statements.md

```diff
@@ -0,0 +1,15 @@
+The most important problems/technologies relevant to Protocol Labs that will/should exist 5-10 years from now:
+
+- certifiable archives (.car)
+- anonymous, private connectivity
+	- distinction between ephemeral and permanent IDs
+- better congestion control
+- libp2p as a service (on your computer)
+- better DHTs
+	- lower latency, more secure
+- packet switching
+	- make libp2p packet oriented instead of stream oriented (currently streams on packets on streams on packets)
```

Comment:
libp2p-as-a-service and packet switching aren't open research problems. We know exactly what to do, it just needs doing